articles/140_topic_and_service_name_mapping.md

```diff
@@ -224,69 +257,73 @@ Here is a non-exhaustive list of prefixes:
 While all planned prefixes consist of two characters, i.e. `rX`, anything proceeding the first namespace separator, i.e. `/`, can be considered part of the prefix.
 The standard reserves the right to use up to 8 characters for the prefix in case additional prefix space is needed in the future.
 
-#### Communicating with Non-ROS Topics
+### Examples of ROS Names to DDS Concepts
```

Comment:
I moved the examples section up and modified the examples to have a table with DDS topic and partition in them.